weblate/addons/base.py

```diff
@@ -278,6 +279,10 @@ def component_update(self, component: Component) -> None:
         """Event handler for component update."""
         # To be implemented in a subclass
 
+    def get_addon_activity_logs(self) -> QuerySet[AddonActivityLog]:
+        """Return activity logs for add-on."""
+        return self.instance.addonactivitylog_set.order_by("-created")
```

Comment:
nit: This method might actually be a better fit for the model class instead of here.